### PR TITLE
added kitchensink

### PR DESCRIPTION
**Pull request by eesha-pamula**

### CODE REVIEW

File: build/kitchensink/install.sh [hunk 1]
```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+#
+#  Mint (C) 2017 Minio, Inc.
```
Comment (harshavardhana) [inline build/kitchensink/install.sh:3]:
```suggestion
#  Mint (C) 2021 Minio, Inc.
```

File: run/core/kitchensink/run.sh [hunk 2]
```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+#
+#  Minio Cloud Storage, (C) 2017 Minio, Inc.
```
Comment (harshavardhana) [inline run/core/kitchensink/run.sh:3]:
```suggestion
#  Minio Cloud Storage, (C) 2021 Minio, Inc.
```

File: run/core/kitchensink/run.sh [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+error_log_file="$2"
+
+BUCKET="my-test-bucket"
+
+(./kitchensink create $SERVER_ENDPOINT $ACCESS_KEY $SECRET_KEY $BUCKET  1>>"$output_log_file" 2>"$error_log_file")
+rv=$?
+if [ "$rv" -ne 0 ]; then
+    exit 1
+    
+./kitchensink verify $SERVER_ENDPOINT $ACCESS_KEY $SECRET_KEY $BUCKET  1>>"$output_log_file" 2>"$error_log_file"
```
Comment (harshavardhana) [inline run/core/kitchensink/run.sh:33]: Are we not deleting the objects after?

**Review by harshavardhana — CHANGES_REQUESTED** [review]

File: run/core/kitchensink/run.sh (hunk 3 above)
Comment (eesha-pamula) [reply run/core/kitchensink/run.sh:33]: I can delete the objects after. I will add that part.

File: run/core/kitchensink/run.sh [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
```
Comment (kannappanr) [inline run/core/kitchensink/run.sh:34]:
```
rv=$?
if [ "$rv" -ne 0 ]; then
    exit 1
```

Comment (kannappanr) [reply run/core/kitchensink/run.sh:34]: need to add this before delete

**Comment by harshavardhana** [comment]
We already incorporated something related to this at https://github.com/minio/minio/tree/master/docs/debugging/s3-check-md5 - used in MinIO CI/CD already.

**Pull request closed without merging**